roapi-http/src/config.rs

```diff
@@ -10,7 +10,8 @@ use std::fs;
 pub struct Config {
     pub addr: Option<String>,
     pub tables: Vec<TableSource>,
-    pub read_only: bool,
+    #[serde(default)]
```

Comment:
this macro attribute was needed to avoid crashes when loading yaml config files that doesn't specify this field. would be good to add a unit test to safe guard this crash in the future and make sure when this field is missing from a yml config, it's always populated with false as the default value.